src/referencePackages/src/microsoft.bcl.asyncinterfaces/1.0.0/Microsoft.Bcl.AsyncInterfaces.1.0.0.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
+    <NuspecFile>$(ArtifactsBinDir)microsoft.bcl.asyncinterfaces/1.0.0/microsoft.bcl.asyncinterfaces.nuspec</NuspecFile>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+   </PropertyGroup>
+
+  <PropertyGroup>
+    <OutputPath>$(ArtifactsBinDir)microsoft.bcl.asyncinterfaces/1.0.0/ref/</OutputPath>
+    <IntermediateOutputPath>$(ArtifactsObjDir)microsoft.bcl.asyncinterfaces/1.0.0</IntermediateOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**/ref/$(TargetFramework)/*.cs" />
+    <Compile Include="**/lib/$(TargetFramework)/*.cs" />
+  </ItemGroup>
+
+    <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+        <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
+    </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
+        <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
+    </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+        <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0-alpha-5" />
```

Comment:
Hmm, could this be deleted because of `<DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>`?

Maybe we should just do a cleanup pass. There's other redundancy in the csproj's, too. Not a problem at the moment because generation is tooled, though.